packages/stores/src/account/osmosis/index.ts

```diff
@@ -1172,6 +1165,63 @@ export class OsmosisAccountImpl {
     );
   }
 
+  makeSplitRoutesSwapExactAmountIn({
+    routes,
+    tokenIn,
+    tokenOutMinAmount,
+  }: {
+    routes: {
+      pools: {
+        id: string;
+        tokenOutDenom: string;
+      }[];
+      tokenInAmount: string;
+    }[];
+    tokenIn: { currency: Currency };
+    tokenOutMinAmount: string;
+  }) {
+    return this.msgOpts.splitRouteSwapExactAmountIn.messageComposer({
+      sender: this.address,
+      routes: routes.map(({ pools, tokenInAmount }) => ({
+        pools: pools.map(({ id, tokenOutDenom }) => ({
+          poolId: BigInt(id),
+          tokenOutDenom: tokenOutDenom,
+        })),
+        tokenInAmount: tokenInAmount,
+      })),
+      tokenInDenom: tokenIn.currency.coinMinimalDenom,
+      tokenOutMinAmount,
+    });
+  }
+
+  makeSwapExactAmountIn({
+    pools,
+    tokenIn,
+    tokenOutMinAmount,
+  }: {
+    pools: {
+      id: string;
+      tokenOutDenom: string;
+    }[];
+    tokenIn: { currency: Currency; amount: string };
+    tokenOutMinAmount: string;
+  }) {
+    return this.msgOpts.swapExactAmountIn.messageComposer({
+      sender: this.address,
+      routes: pools.map(({ id, tokenOutDenom }) => {
+        return {
+          poolId: BigInt(id),
+          tokenOutDenom: tokenOutDenom,
+        };
+      }),
+      tokenIn: {
+        denom: tokenIn.currency.coinMinimalDenom,
+        amount: tokenIn.amount.toString(),
+      },
+      tokenOutMinAmount,
+    });
+  }
```

Comment:
NIT: what is the rationale behind splitting these into functions? Are they meant to be reused? I ask because they don't over any functionality other than directly calling and returning the message composers.